src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Added `Controller::json` to simplify creating JSON responses when using the Serializer component
  * Deprecated absolute template paths support in the template name parser
+ * Deprecated services for core form types and core form type extensions without dependencies
```

Comment:
this is still wrong for type extensions. Type extensions always need to be registered